great_expectations/data_context/data_context/base_data_context.py

```diff
@@ -260,6 +264,18 @@ def __init__(
                     validation_operator_config,
                 )
 
+    @staticmethod
+    def _resolve_BaseDataContext_cloud_args(
```

Comment:
Since this is a static method on BaseDataContext, we can probably remove that reference from the name

Reply:
was getting some type errors but I think referencing with the class name instead of `self` should resolve this.